practice/src/css/layouts/header.css

```diff
@@ -0,0 +1,166 @@
+.header-section {
+  padding: 16px 32px;
+}
+  
+.header-logo {
+  display: flex;
+  align-items: center;
+  gap: 4px;
+}
+
+.header-name {
+  text-decoration: none;
+  font-family: var(--secondary-font);
+  font-weight: var(--font-weight-regular);
+  line-height: 20px;
+  color: var(--secondary-text-color);
+  font-size: var(--font-size-xsmall);
+}
+  
+.header-nav {
+  display: grid;
+  grid-template-columns: 1fr auto;
+  align-items: center;
+  gap: 10px;
+}
+  
+label {
+  cursor: pointer;
+}
+  
+#header-dropdown {
+  display: none;
+}
+  
+.nav-list {
+  display: none;
+}
+  
+#header-dropdown:checked + .nav-list {
+  grid-column: span 2;
+  display: grid;
+  gap: 15px;
+  text-align: center;
+  background: rgba(70, 107, 243, 0.1);
+  border-radius: 12px;
+}
+  
+.header-button {
+  display: none;
+}
+
+.btn-login {
+  background: rgba(70, 107, 243, 0.1);
```

Comment:
Should create a variable for this color. Because I see that you use this color many times